task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -369,7 +378,7 @@ def _close_unused_sockets(*sockets):
 
     def _send_startup_message(self, ti: TaskInstance, path: str | os.PathLike[str], child_comms: socket):
         """Send startup message to the subprocess."""
-        msg = StartupDetails(
+        msg = StartupDetails.model_construct(
```

Comment:
Why this change? for skipping validation?

Reply:
Yeah.